sportnet/templates/includes/inc_navbar_not_registered.html

```diff
@@ -20,8 +20,9 @@
                 <form action="#">
                     <input class="uk-input uk-margin-small" type="text" placeholder="email">
                     <input class="uk-input uk-margin-small uk-margin-remove-top" type="password" placeholder="password">
-                    <a class="uk-button uk-button-primary uk-width-1-1 uk-margin-bottom">Войти</a>
-                    <a class="uk-button uk-button-primary uk-width-1-1 uk-margin-bottom">Регистрация</a>
+                    <a href="#" class="uk-button uk-button-primary uk-width-1-1 uk-margin-bottom">Войти</a>
+                    <a href="{% url 'account_reset_password' %}" class="uk-button uk-button-danger uk-width-1-1 uk-margin-bottom">Войти</a>
```

Comment:
Предполагаю, что это кнопка сброса, а не входа
```suggestion
                    <a href="{% url 'account_reset_password' %}" class="uk-button uk-button-danger uk-width-1-1 uk-margin-bottom">Восстановить пароль</a>
```

Reply:
поправил